Viper/Viper/HotelOfCountry/HotelTableViewController.swift

```diff
@@ -0,0 +1,58 @@
+// HotelTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+final class HotelTableViewController: UITableViewController {
+    // MARK: - Public properties
+
+    var presenter: HotelsPresenterProtocol?
+
+    // MARK: - Life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUI()
+    }
+
+    // MARK: - Private properties
+
+    private func setUI() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(HotelTableViewCell.self, forCellReuseIdentifier: "hotelCell")
```

Comment:
```suggestion
        tableView.register(HotelTableViewCell.self, forCellReuseIdentifier: "")
```
в константы свифтген